webapp/client/src/components/Menu/menu.js

```diff
@@ -64,10 +65,26 @@ class Landing extends Component {
     page ? window.pause() : window.resume()
   }
 
+  overrideLightModeOnDashboardScreens = () => {
+    if (!this.props.appState.showGame) {
+      document.body.style.background = "#202020"
+      document.getElementById("logo").src = Logo_dark;
+    }
+  }
+
+  resumeLightModeIfOn = () => {
+    if (!window.darkMode) {
+      document.body.style.background = "#FFFFFF"
+      document.getElementById("logo").src = Logo_light;
+    }
+  }
+
   resumeGame = () => {
     if (window.gamePaused && window.gameHasBegun) {
       window.resume()
       this.state.history.push('/game')
+      this.props.appState.set({ showGame: true }) 
```

Comment:
... this is a weird react thing, and a weird method I built to handle the weird thing. but whenever changing any of the  states for any elements in the component, you actually have call a the setState method to change the 
 value throughout the component(s)... So like this > `this.setState({ some_var: "new value" })` But, that'll only change the state of the component that you're in. And, I often want to change site-wide value. so the appState.set() essentially relays to the highest parent component to change all the "ShowGame" references through the site....

... So, you pretty much dove into the most complicated aspect of this site.